### ISSUE TITLE
Node: TChannelAsThrift.send, logger is undefined

### ISSUE THREAD

**Issue by Jeloi** [uber/tchannel#924]
`self.logger = self.logger || request.logger;`

Should be `request.channel.logger` ?

Needs to be updated after request was refactored

**Comment by Raynos** [comment]
Correct; it should now be `request.channel.logger` which was broken due to a refactor.
